src/Tools/dotnet-dev-certs/src/Program.cs

```diff
@@ -378,8 +368,9 @@ private static int EnsureHttpsCertificate(CommandOption exportPath, CommandOptio
 
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
                 {
-                    reporter.Warn("Trusting the HTTPS development certificate was requested. Trusting the certificate on Linux distributions automatically is not supported. " +
-                        "For instructions on how to manually trust the certificate on your Linux distribution, go to https://aka.ms/dev-certs-trust");
+                    reporter.Warn("Trusting the HTTPS development certificate was requested. If the certificate is not " +
+                        "already trusted we will run commands using 'sudo'." +
+                        Environment.NewLine + "This might prompt you for your password.");
```

Comment:
This check, as well as the one below needs to be based on the combination of distro and version. We don't want to "onboard" distros/versions by default without having tested them explicitly and created the infrastructure to prevent regressions.

We should keep the message for unsupported distro/vesions.

Reply:
I've added the message back, it gets printed when there is no way of trusting certificates on the platform.